docs/README.md

```
@@ -1,41 +1,44 @@
 
 ```bash
-USE_SSH=true yarn deploy
+GIT_USER=<your-github-username> yarn deploy
 ```
 
-Not using SSH:
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
 
-```bash
-GIT_USER=<Your GitHub username> yarn deploy
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)